.github/workflows/release.yml

```diff
@@ -129,7 +129,7 @@ jobs:
       - name: Get the build number
         run: |
           export VERSION="${CONTAINER_TAG}"
-          export BUILD_NUMBER=$((`curl -s https://quay.io/api/v1/repository/skupper/skupper-router/tag/?filter_tag_name=like:${VERSION} | jq .tags[].name | grep -E "\"${VERSION}-[0-9]+\"" | wc -l`))
+          export BUILD_NUMBER=$((`curl -s https://quay.io/api/v1/repository/skupper/skupper-router/tag/?filter_tag_name=like:${VERSION} | jq '.tags[] | select(has("end_ts") | not) | .name' | grep -E "\"${VERSION}-[0-9]+\"" | wc -l || echo 0` + 1))
```

Comment:
This last sentence was not incrementing the build number before.
I believe you should stop after `wc -l` as done before, removing ` || echo 0` + 1`.

Reply:
I fixed it ! Thanks. I did not realize that BUILD_NUMBER in this case is calculated after the build is complete